src/CppAst/CppTypedef.cs

```diff
@@ -20,8 +21,15 @@ public CppTypedef(string name, CppType type) : base(CppTypeKind.Typedef)
         {
             Name = name ?? throw new ArgumentNullException(nameof(name));
             ElementType = type;
+            Attributes = new List<CppAttribute>();
+            TokenAttributes = new List<CppAttribute>();
         }
 
+        public List<CppAttribute> Attributes { get; }
+
+        [Obsolete("TokenAttributes is deprecated. please use system attributes and annotate attributes")]
```

Comment:
Not related to your PR, I am not really a big fan of this obsolete attribute given that the code base is still using it all over the place but it does not look like there's a lot we can do.

Reply:
I thought I removed that, it came in from the code I pulled in from the main branch. I'll remove it because I agree, it's annoying to have in Bindr.